zerver/webhooks/gogs/doc.html

```diff
@@ -0,0 +1,44 @@
+<p>
+    First, create the stream you'd like to use for Gogs notifications, and then
+    subscribe all interested parties to this stream. The integration will use
+    the default stream <code>commits</code> if no stream is supplied in the
+    hook; you still need to <a target="_blank" href="../#streams/new">create the
+        stream</a> even if you are using this default.
+</p>
+
+<p>Next, go to your repository page and click <b>Settings</b>:</p>
+<p><img src="/static/images/integrations/gogs/001.png"/></p>
+
+<p>From there, select <b>Webhooks</b>:</p>
```

Comment:
I think we can be more specific by saying:

```html
<p>On the **Settings** page, select the <b>Webhooks</b> link on the left sidebar to go to the **Webhooks** page:</p>
```

(I'm not familiar with Gogs, you might want to edit this if some parts are incorrect)